Xamarin.Forms.Platform.Android/Renderers/ListViewAdapter.cs

```diff
@@ -44,6 +45,11 @@ internal class ListViewAdapter : CellAdapter
 		IListViewController Controller => _listView;
 		protected ITemplatedItemsView<Cell> TemplatedItemsView => _listView;
 
+		internal ListViewAdapter(IntPtr handle, JniHandleOwnership ownership) : base(handle, ownership)
+		{
+			throw new Exception("Unexpected ListViewAdapter ressurection.");
```

Comment:
If Android attempts to resurrect the adapter in the future, then it'll hit this exception which throws a preferable excdeption to a missing ctor exception.